clojure/src/firn/layout.clj

```diff
@@ -160,6 +160,9 @@
      :site-map      (config :site-map)
      :site-links    (config :site-links)
      :site-logs     (config :site-logs)
+     :site-title    (config :site-title)
+     :site-author   (config :site-author)
+     :site-desc     (config :site-desc)
```

Comment:
This is great! Do you mind  updating the [docs](https://firn.theiceshelf.com/data-and-metadata#metadata) table to match the updated layout vals?

Reply:
Updated.

Reply:
FYI, the templates for the docs site can also be updated to include a title, author, and description.